app/lib/modules/llm/providers/ollama.ts

```diff
@@ -63,7 +81,7 @@ export default class OllamaProvider extends BaseProvider {
        * Running in Server
        * Backend: Check if we're running in Docker
        */
-      const isDocker = process?.env?.RUNNING_IN_DOCKER === 'true' || serverEnv?.RUNNING_IN_DOCKER === 'true';
+      const isDocker = serverEnv?.RUNNING_IN_DOCKER === 'true';
```

Comment:
same goes here  `process?.env?.RUNNING_IN_DOCKER` please keep this as it is. its used o=in non cloudflare environment